solutions/usaco-896.mdx

```diff
@@ -125,4 +125,31 @@ class mountain implements Comparable<mountain> {
 
 </JavaSection>
 
+<PySection>
+	
+```python
+use = []
+	
+with open('mountains.in') as read:
+	n = int(read.readline())
+	for _ in range(n):
+		a, b = map(int, read.readline.split())
+		use.append((a - b, a + b))
+
+use.sort(key=lambda x: (x[0], -x[1]))
+cur = use[0][1]
+
+res = 1
```

Comment:
either use `ans` like the other sols or just use a more informative var name

Reply:
res stands for result? I'll change it to ans

Reply:
yeah but the other sols use `ans`

Reply:
i will change it for you. <3

Reply:
done